src/resources/views/crud/columns/inc/wrapper_end.blade.php

```diff
@@ -9,4 +9,8 @@
     }
 @endphp
 
+@if($column['wrapper']['prefix'] ?? false)
```

Comment:
Wait. It says `prefix` here and `suffix` below... Is this correct?!

Reply:
Good catch! 👀
Indeed, it's incorrect.

Reply:
```suggestion
@if($column['wrapper']['suffix'] ?? false)
```

Reply:
@tabacitu I did the https://github.com/Laravel-Backpack/CRUD/pull/3502/commits/d77c3086f089b7f1d93206f562422316e3ffae7d commit directly from github interface, to fix the issue in this thread. And suddenly CI bot did a StyleCI fix on files that were not commited 😬

Now there are 71 changes and some conflicts with master.

_I know I should be fixing this, but I was organizing my notifications and this was an easy one 😅_

Reply:
Merged the bogus changes on `master`. Merged `master` into this branch. Seems to be ok now... 🤷‍♂️ Let's see..

Reply:
Great 👌